cg/models/report/validators.py

```diff
@@ -33,14 +33,18 @@ def get_boolean_as_string(value: bool | None) -> str:
     return NA_FIELD
 
 
-def get_float_as_string(value: float | None) -> str:
-    """Return string representation of a float value."""
-    return str(round(float(value), PRECISION)) if value or isinstance(value, float) else NA_FIELD
+def get_number_as_string(value: int | float | None) -> str:
+    """Return string representation of a number."""
+    return (
+        str(round(float(value), PRECISION))
+        if value or isinstance(value, (int, float))
```

Comment:
Im wondering what would happen if the input is bool (True) or str, would the behaviour be unexpected?
```suggestion
        if isinstance(value, (int, float))
```

Reply:
If bool or an invalid string (not coercible to a float) are passed we would like this to fail actually.

Reply:
just edited the comment as I realised some coverage values are parsed as stirngs for mip :( 

>  I will keep the value check as some fields may be parsed as strings